openfl/interface/interactive_api/experiment.py

```diff
@@ -61,27 +70,34 @@ def get_best_model(self):
         tensor_dict = self.federation.dir_client.get_best_model(
             experiment_name=self.experiment_name)
 
-        return self._rebuild_model(tensor_dict)
+        return self._rebuild_model(tensor_dict, model_status=ModelStatus.BEST)
 
     def get_last_model(self):
         """Retrieve the aggregated model after the last round."""
         self._assert_experiment_accepted()
         tensor_dict = self.federation.dir_client.get_last_model(
             experiment_name=self.experiment_name)
 
-        return self._rebuild_model(tensor_dict)
+        return self._rebuild_model(tensor_dict, model_status=ModelStatus.LAST)
 
-    def _rebuild_model(self, tensor_dict):
+    def _rebuild_model(self, tensor_dict, model_status=ModelStatus.BEST):
         """Use tensor dict to update model weights."""
         if len(tensor_dict) == 0:
-            self.logger.error('No tensors received from director')
-            self.logger.error(
-                'Possible reasons:\n'
-                '1. Aggregated model is not ready \n'
-                '2. Experiment data removed from director'
-            )
+            warning_msg = ('No tensors received from director\n'
+                           'Possible reasons:\n'
+                           '\t1. Aggregated model is not ready\n'
+                           '\t2. Experiment data removed from director')
+
+            if model_status == ModelStatus.BEST and not self.validation_task_exist:
+                warning_msg += '\n\t3. No validation tasks are provided'
+
+            warning_msg += f'\nReturn {self.current_model_status} model'
```

Comment:
`\n` - should be in the end of the string

Reply:
Why? It wasn't needed before https://github.com/intel/openfl/blob/544c811dd61e93e919ec7e7791452cc84d1c4b03/openfl/interface/interactive_api/experiment.py#L81